run-test.py

```diff
@@ -851,13 +851,13 @@ def deploy_observer(path, keep_database=False):
             "/../observer && GO111MODULE=on make migrate")
     # run observer
     ssh(OBSERVER, """tmux new-session -d -s observer \\"cd """+INSPATH +
-        """/../observer && ./bin/observer 2>&1 | tee -a observer.log; bash\\" """)
+        """/../observer && ./bin/observer ./artifacts/observer.yaml 2>&1 | tee -a observer.log; bash\\" """)
```

Comment:
we are setting path to config without `--config` flag, aren't we?

Reply:
also, path should be `.artifacts/observer.yam`l